src/sentry/data/samples/transaction.json

```diff
@@ -145,11 +145,71 @@
          "trace_id": "a7d67cf796774551a95be6543cacd459",
          "op": "db",
          "data": {},
-         "span_id": "b048b4c8fdc5168c"
+         "span_id": "b048b4c8fdc5168c",
+         "start_timestamp": 1569876654.900,
+         "timestamp": 1569876654.910
+      },
+      {
+         "same_process_as_parent":true,
+         "description": "Django: SELECT \"countries\".\"id\", \"countries\".\"name\", \"countries\".\"continent\", \"countries\".\"region\", \"countries\".\"surface_area\", \"coun...'CAN'",
+         "tags": {
+            "error":false
+         },
+         "parent_span_id": "b048b4c8fdc5168c",
+         "trace_id": "a7d67cf796774551a95be6543cacd459",
+         "op": "db",
+         "data": {},
+         "span_id": "0beec7b5ea3f0fdb",
+         "start_timestamp": 1569876654.910,
+         "timestamp": 1569876654.911
+      },
+      {
+         "same_process_as_parent":true,
+         "description": "Django: SELECT \"countries\".\"id\", \"countries\".\"name\", \"countries\".\"continent\", \"countries\".\"region\", \"countries\".\"surface_area\", \"coun...'CAN'",
+         "tags": {
+            "error":false
+         },
+         "parent_span_id": "0beec7b5ea3f0fdb",
+         "trace_id": "a7d67cf796774551a95be6543cacd459",
+         "op": "db",
+         "data": {},
+         "span_id": "c95d0dd47f3c5bc2",
+         "start_timestamp": 1569876654.912,
+         "timestamp": 1569876654.914
+      },
+      {
+         "same_process_as_parent":true,
+         "description": "Django: SELECT \"countries\".\"id\", \"countries\".\"name\", \"countries\".\"continent\", \"countries\".\"region\", \"countries\".\"surface_area\", \"coun...'CAN'",
+         "tags": {
+            "error":false
+         },
+         "parent_span_id": "c95d0dd47f3c5bc2",
+         "trace_id": "a7d67cf796774551a95be6543cacd459",
+         "op": "unknown op",
+         "data": {},
+         "span_id": "62cdb7020ff920e5",
+         "start_timestamp": 1569876654.914,
+         "timestamp": 1569876654.919
+      },
+      {
+         "same_process_as_parent":true,
+         "description": "Django: SELECT \"countries\".\"id\", \"countries\".\"name\", \"countries\".\"continent\", \"countries\".\"region\", \"countries\".\"surface_area\", \"coun...'CAN'",
+         "tags": {
+            "error":false
+         },
+         "parent_span_id": "babaae0d4b7512d9",
+         "trace_id": "a7d67cf796774551a95be6543cacd459",
+         "op": "http",
+         "data": {},
+         "span_id": "bbe960a25ea311d2",
+         "start_timestamp": 1569876654.913,
+         "timestamp": 1569876654.920
       }
    ],
    "transaction": "/country_by_code/",
    "type": "transaction",
+   "start_timestamp": 1569876654.900,
+   "timestamp": 1569876654.920,
```

Comment:
One reason this event didn't have timestamps before is that the `load_data()` function added them. With a fixed timestamp this event will become invalid in 30 days.

If you store the number of ms from the event start then load_data() could generate a consistent span waterfall by adding the new start time.

Reply:
I'll try this out.

Reply:
@markstory do you have recommendations for injecting timestamps for the individual descendent spans?

Reply:
I've since added this.